ROADMAP.md

```diff
@@ -1,7 +1,18 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.7 Release - planning now in process 
-* Please join the Kubeflow Community Meetings on Tuesdays for updates and for opportunities to contribute.  
+## Kubeflow 1.7 Release, Planned: March 2023 
+The Kubeflow Community plans to deliver its v1.7 release in March 2023, per this [timeline](https://github.com/kubeflow/community/pull/573).   The high level deliveries are being tracked in this [Project Board](https://github.com/orgs/kubeflow/projects/50/views/1).   The v1.7 release process will be managed by the v1.7 [Release Team](https://github.com/kubeflow/internal-acls/pull/576) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md)
+
+Notable feature candidates in the [Project Board](https://github.com/orgs/kubeflow/projects/50/views/1) are:
+* Support for Kubernetes 1.25
+* Improved user isolation especially for the Kubeflow pipelines user interface, database, and artifacts
+* Update Kubeflow Noteboks naming from Notebooks to Workbenches
+* Delivery of KFP V2 beta with its new front-end, backend and SDK 
+* New, time saving workflows to simplify data exchange from Katib to Kubeflow Pipelines
```

Comment:
I think, we should remove this part for Katib also.

Maybe we can add the following:
- Simplified creation of Katib and Training Operator experiments using SDKs.

Any other items that we certainly will deliver in KF 1.7 @kubeflow/wg-training-leads @tenzen-y @anencore94 ?

Reply:
@andreyvelich I have incorporated this update.

Reply:
I have no items.

Reply:
@jbottum We discussed on today's AutoML community meeting that we want to promote a few Katib UI changes as part of Kubeflow 1.7 ROADMAP. 
Can we indicate the most significant improvements/changes for the user that we will deliver in 1.7 ?
@kimwnasptd @johnugeorge What Katib UI changes do we want on the ROADMAP that we deliver ?

Reply:
Expose trial logs(https://github.com/kubeflow/katib/issues/971 )
Filtering, Sorting trial list(https://github.com/kubeflow/katib/issues/1441 )

Reply:
@andreyvelich @johnugeorge I have added line 13 to include these feature candidates in the roadmap.   Please provide looks good or comments.   thanks.

Reply:
Thank you @jbottum. LGTM

Reply:
LGTM